src/js/helpers/aboutTheDataHelper.js

```diff
@@ -2,10 +2,65 @@
  * aboutTheDataHelper.js
  * Created by Jonathan Hill 11/20/20
  */
+import { useState } from 'react';
+import { stringify } from 'query-string';
```

Comment:
This library should be removed from our package.json. Node has it's own query string module.

https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options